src/mmw/apps/modeling/migrations/0023_fix_gis_data_serialization.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import ast
+import json
+
+from django.db import migrations
+
+
+def fix_gis_data_serialization(apps, schema_editor):
+    Project = apps.get_model('modeling', 'Project')
+    for project in Project.objects.all():
```

Comment:
There are 18,574 projects in prod & 933 in staging. Can you anticipate how the migration will might perform in those environments?  My suspicion is that we don't need to run this migration in prod, since the initiating commit hasn't been deployed, but I'm not sure there is a good mechanism to skip it, other than manually inserting the migration record so it appears to have run.

Reply:
I just ran this query on staging:

```sql
SELECT _left, COUNT(_left)
FROM (
    SELECT LEFT(gis_data, 10) AS _left
    FROM modeling_project
    WHERE gis_data IS NOT NULL) AS x
GROUP BY _left;

   _left    | count
------------+-------
 {"n38": 0, |    80
 {u'n38': 0 |    14
(2 rows)
```

Not sure which one is which, but we can probably use a similar test to significantly limit the operation.

Reply:
I think this is old:

```sql
SELECT LEFT(gis_data, 10) AS _left
FROM modeling_project
WHERE gis_data IS NOT NULL
ORDER BY created_at ASC
LIMIT 5;

   _left
------------
 {"n38": 0,
 {"n38": 0,
 {"n38": 0,
 {"n38": 0,
 {"n38": 0,
(5 rows)
```

and this is new:

```sql
SELECT LEFT(gis_data, 10) AS _left
FROM modeling_project
WHERE gis_data IS NOT NULL
ORDER BY created_at DESC
LIMIT 5;

   _left
------------
 {u'n38': 0
 {u'n38': 0
 {u'n38': 0
 {u'n38': 0
 {u'n38': 0
(5 rows)
```

Note: the sort order is opposite in the above queries.

Reply:
I pushed a FIXUP commit that alters the migration to be more efficient, and will make it a noop in production, where there are no incorrectly serialized records.